packages/utils/src/index.js

```diff
@@ -110,6 +110,9 @@ export const zipObj = obj =>
 // compose([f, g, h])(o) = h(g(f(o)))
 export const compose = fns => o => fns.reduce((acc, fn) => fn(acc), o);
 
+// composePlugins([f, g, h])(o, e) = h(g(f(o, e), e), e)
+export const composePlugins = fns => (o, e) => fns.reduce((acc, fn) => fn(acc, e), o);
```

Comment:
I think we'd be better off having this function defined inline in `Keystone/index.js`, as that's the only place it's ever really going to be used, as opposed to the more general `compose` function. In principle (although not quite always in practice) we want to keep the `utils` package generic enough that it could be used by other projects that aren't keystone, and this function is just a bit too keystone specific. Putting the function inline also means we only need to have a `minor` version bump on the keystone package, since it's a backwards compatible change.

Reply:
done